tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "ES2017",
     "module": "CommonJS",
     "moduleResolution": "Node",
-    "lib": ["ES2019"],
+    "lib": ["ES2019", "DOM"],
```

Comment:
This is currently required because of https://github.com/evanw/esbuild/issues/2388